flask/homework.py

```diff
@@ -0,0 +1,28 @@
+from flask import Flask, render_template
+
+app = Flask(__name__)
+
+
+#vegetables = ['beans', 'carrot', 'beetroot', 'cucumber']
+#fruits = ['melon', 'apple', 'strawberry', 'grape']
+
+
+@app.route('/')
+def home():
+    return render_template('home.html')
+
+
+@app.route('/vegetables')
+def vegetables():
+    vegetables_list = ['beans', 'carrot', 'beetroot', 'cucumber']
```

Comment:
if you never gonna change this list in your code, use tuple

Reply:
modif to tuple